### PR TITLE
CO-3049 Muskathlon migration v11

### PR DESCRIPTION
**Pull request by josueBulle**
Merge first:

website_event_compassion
quentin/partner_compassion
quentin/sponsorship_switzerland
quentin/report_compassion
quentin/child_sync_wp
quentin/sms_939
quentin/ogone
quentin/child_switzerland
quentin/partner_communication

Various fixes on [compassion_modules](https://github.com/CompassionCH/compassion-modules/pull/1288)

### CODE REVIEW

File: muskathlon/models/advocate_details.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
Comment (ecino) [inline muskathlon/models/advocate_details.py:1]: You should make inherit the class from GeoModel

**Review by ecino — CHANGES_REQUESTED** [review]
```
muskathlon/controllers/main.py:10:22: E401 multiple imports on one line
```
Can you rebase your branch onto 11.0 ? I think some travis errors are because you have an old version.

**Comment by josueBulle** [comment]
I don't understand Travis errors...
There are a lot of "ConnectionRefusedError: [Errno 111] Connection refused"

**Comment by ecino** [comment]
> I don't understand Travis errors...
> There are a lot of "ConnectionRefusedError: [Errno 111] Connection refused"

Your branch is behind v11. It runs the tests of the other modules which are in an older version and I think we had some mistakes previously. Maybe it will fix the errors if you rebase properly so that you commits sit on top of the current 11.0 state.

image

**Comment by josueBulle** [comment]
I did rebase when you asked yesterday. In your screenshot, it is 10 commits behind v10. It is up to date with branch v11.
Muskathlon has no test... Idk what I did/understood wrong

**Comment by josueBulle** [comment]
I included two fixes for already-merged migrations instead of creating a new PR

**Review by ecino — APPROVED** [review]

**Pull request merged**